src/main.rs

```diff
@@ -53,6 +54,9 @@ use x509::time::{Time, Validity};
 use x509::{Certificate, TbsCertificate};
 use zeroize::Zeroizing;
 
+const PKCS10: &str = "application/pkcs10";
+const PKCS10_BUNDLE: &str = "application/pkcs10-bundle";
```

Comment:
This is not a registered media type, so we'll have to define a custom vendor type for it.

This should look something like this https://github.com/profianinc/drawbridge/blob/26f5e3d127c1b814d31ce8cbec939cc8932c4c5a/crates/type/src/tree/directory.rs#L19-L21, but with `steward` as the domain, not drawbridge

Reply:
`application/vnd.steward.pkcs10-bundle.v1`

Should work fine

Reply:
Done